diff --git a/data/org.midori_browser.Midori.appdata.xml.in b/data/org.midori_browser.Midori.appdata.xml.in
index d318894d..f1dbfbeb 100644
--- a/data/org.midori_browser.Midori.appdata.xml.in
+++ b/data/org.midori_browser.Midori.appdata.xml.in
@@ -27,9 +27,6 @@
       <li>Adblock filter list support.</li>
       <li>Private browsing.</li>
       <li>Manage cookies and scripts.</li>
-    </ul>
-    <p><b></b></p>
-    <ul>
       <li>Open a 1000 tabs instantly.</li>
       <li>Easy web apps creation.</li>
       <li>Customizable side panels.</li>
